documentation/modules/ROOT/partials/modules/all-connectors/ref-connector-monitoring-snapshot-metrics.adoc

```diff
@@ -18,11 +18,6 @@
 |`long`
 | The number of events that have been filtered by include/exclude list filtering rules configured on the connector.
 
-|[[connectors-snaps-metric-monitoredtables_{context}]]<<connectors-snaps-metric-monitoredtables_{context}, `MonitoredTables`>> +
```

Comment:
@roldanbob, what's your take on this? For upstream, we actively undocument deprecated options -- i.e. they are still available in the code, but we don't mention them any longer, and in some cases using them may also trigger a warning. Can we do the same for downstream docs, or do you prefer to keep deprecated things listed there?

Reply:
For the downstream docs we should do the same, though it's probably good to drop information in a release note to mention when we remove an option.  That might head off questions to Support -- or at least give them an RTFM option if a question does pop up. 
Is there someplace that I can find information about items that we deprecate in a given release?

Reply:
@gunnarmorling @ani-sha I had a conversation today about this with @fbolton. I wasn't aware, but, downstream, we have a requirement to notify customers at least one release in advance of removing a deprecated feature. So since, as you say, the option remains in the code for now, downstream we should continue to include it in the documentation, clearly designated as being deprecated. We should also add a release note to reinforce that point.  In the next downstream release, when the option is removed, I can follow up with a release note to state that the option is no longer available.

So, for now, I'd request that we retain the content upstream with a deprecation statement, or, at least retain it in the source, conditionalized as `[product]` content (again with the deprecation statement).

Reply:
Ok, thanks for clarifying, @roldanbob! So,

@ani-sha, could you rework this change, so to keep the deprecated option but add the right "if-def product" tags?

@roldanbob, could you coordinate with @fbolton to add this release note and also log a DBZ Jira issue for tracking that follow-up activity for the next release?

Thanks!

Reply:
Added Jira issue [DBZ-3768](https://issues.redhat.com/browse/DBZ-3768) to track the requirement for a release note in the next downstream release.